strider/server.py

```diff
@@ -159,11 +159,18 @@ def get_finished_query(
     results.expire(expiration_seconds)
     logs.expire(expiration_seconds)
 
+    # strip categories from node bindings
+    results = list(results.get())
+    for result in results:
+        for bindings in result["node_bindings"].values():
+            for binding in bindings:
+                binding.pop("category")
```

Comment:
Should this be `categories`?

Reply:
938a893c352b0720e110031a0615fc7969eca083